packages/webapp/src/pages/index.tsx

```diff
@@ -37,6 +38,8 @@ const Home: FablePage = ({ isHydrated }) => {
   const isRightNetwork = !notConnected && chainSupported
   const isWrongNetwork = !notConnected && !chainSupported
 
+    useOfflineCheck();
+
```

Comment:
Sorry I forgot this one is the last review, could you add this to `_app.tsx` instead, so it applies to all pages?

Reply:
sure